src/index.html

```diff
@@ -23,11 +24,42 @@
 
     </head>
     <body class="flex-container" x-data="carFestival()">
-        <div class="excuse-top">I need a new car because ...</div>
+        <div class="excuse-top"><h1>I need a new car because ...</h1></div>
         <div class="excuse-bottom" x-show="!showGuessButton" x-text="currentExcuse"></div>
         <div class="generate-button-container" x-show="showGuessButton">
             <button type="button" class="generate-button" x-on:click="guess">Generate excuse</button>
         </div>
+
+        <div class="generate-button-container">
```

Comment:
The "problem" with this div (and the one you added below with id="infobox"), is that their pre-existing siblings (the divs with class excuse-top, and either one of excuse-bottom OR generate-button-container) are set to have 50% of the viewport height each. Which means that excuse-top + (excuse-bottom OR generate-button-container) = already 100% of the viewport height.

After you add the additional generate-button-container (which, remember, has 50% of vh) and infobox divs, this results in a visually glitchy looking design (elements seem to be oddly placed).

Now there's a multitude of options to fix this (including making the base design more generic), but since your infobox seems to be intended to be like some kind of modal, you could use a combination of relative/absolute or fixed positioning (allowing you to place it over content), see for example https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/

Reply:
I will create a better modal CSS for this. I have tested the page in a mobile preview, and ttbt it looks not great even without my box.